tachyon/base/buffer/copyable_forward.h

```diff
@@ -11,7 +11,7 @@ class Buffer;
 class ReadOnlyBuffer;
 
 // NOTE: Do not implement for builtin serializable.
-// See tachyon/base/buffer.h
+// See tachyon/base/buffer/read_only_buffer.h
```

Comment:
Shouldn't this change be with ec00d5e?

Reply:
I meant it to be `tachyon/base/buffer/buffer.h` because `tachyon/base/buffer.h` is non-existent path.